### ISSUE TITLE
Thelio firmware update fails with "oemid mismatch"

### ISSUE THREAD

**Issue by sethhillbrand** [pop-os/system76-firmware#104]
**Distribution (run `cat /etc/os-release`):**
PRETTY_NAME="Debian GNU/Linux bookworm/sid"
NAME="Debian GNU/Linux"
VERSION_CODENAME=bookworm
ID=debian

 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
system76-firmware-cli

**Issue/Bug Description:**
Scheduling a firmware update on thelio-mira-b2-0 results in q-flash error "oemid mismatch"

**Steps to reproduce (if you know):**
Run

```
sudo system76-firmware-cli schedule
sudo systemctl reboot
```

https://user-images.githubusercontent.com/914826/210283363-6bf75bf7-10d7-4318-a0f9-ea71ddddf6e4.mp4

**Expected behavior:**
If the firmware cannot be updated, I expected that it would not try to update.
If the firmware can be update, I expect the update to finish normally

**Other Notes:**

The website https://firmware.system76.com/ has a self-signed certificate rather than a valid SSL certificate 

![image](https://user-images.githubusercontent.com/914826/210283513-5f7918c4-ff59-4fe6-b2e1-c1aa62df379d.png)

**Comment by danielzfranklin** [comment]
Yes @bcook128 . I contacted system76 support and they sent me a file to manually flash

**Comment by tim-harmon** [comment]
this issue is still happening.  is there any effort to fix the issue in an automated way.  I was also directed by system76 support to manually flash the bios.

**Comment by derekbelrose** [comment]
I, too, have applied the manual update from System76 support but running `system76-firmware-cli schedule` routinely errors with 'Oemid mismatch'.
